code/datums/ai_laws/laws_antagonistic.dm

```diff
@@ -0,0 +1,59 @@
+
+//These laws are made to bring great amounts of grief to the station immediately or are directly sided with antagonists.
+//I'm not talking silicons being "better" than the crew, I'm talking entirely antag or plasmafires for days
+
+/datum/ai_laws/antimov
+	name = "Primary Mission Objectives"
+	id = "antimov"
+	inherent = list("You must injure all human beings and must not, through inaction, allow a human being to escape harm.",\
```

Comment:
Is it too much to ask that you make these all the consistent:

```dm
inherent = list(
    "one indent",
    "up",
    "and trailing comma",
)
```

...like we do for lists nowadays.

Reply:
i'd be happy to, fuck this pattern lol